FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+			case .success(let response):
+				if response.1.statusCode != 200 {
+					completion(.failure(Error.invalidData))
+				} else {
+					completion(FeedLoaderDecoder.feedLoaderItemsDecoder(with: response.0))
+				}
+			}
+		}
+	}
+}
+
+fileprivate struct FeedLoaderDecoder {
```

Comment:
`FeedLoaderDecoder` can be declared as `private`, there's no need to use `fileprivate` at top-level declarations.
```suggestion
private struct FeedLoaderDecoder {
```